projects/archiver/src/tasks/indexer/index.ts

```diff
@@ -57,27 +57,30 @@ const handlerCurry: (
             const all = [thisIssueSummary, ...otherIssuesSummariesForEdition]
             const allSortedEditionsIssues = issueSummarySort(all)
 
-            await upload(
-                `${issuePublication.edition}/issues`,
-                allSortedEditionsIssues,
-                Bucket,
-                'application/json',
-                FIVE_SECONDS,
-            )
-
-            // Also upload the index into the root for older clients
-            // TODO: this can be removed once we are happy that the clients are consuming the namespaced index
-            if (issuePublication.edition === 'daily-edition') {
+            // make sure we don't upload an empty list of editions
+            if (allSortedEditionsIssues.length > 0) {
```

Comment:
if the length is zero we should get an alarm, though?

Reply:
I mean, we should set up an alarm so we know when things are going wrong.

Reply:
I've added some logging for this situation. I could add some kind of pagerduty integration in a separate PR